gamestonk_terminal/economy/economy_controller.py

```diff
@@ -71,45 +122,42 @@ def __init__(self):
     @staticmethod
     def print_help():
         """Print help"""
-        print(
-            "https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/gamestonk_terminal/economy"
-        )
-        print("")
-        print(">> ECONOMY <<")
-        print("")
-        print("What do you want to do?")
-        print("   cls           clear screen")
-        print("   ?/help        show this menu again")
-        print("   q             quit this menu, and shows back to main menu")
-        print("   quit          quit to abandon program")
-        print(" ")
-        print("   feargreed     CNN Fear and Greed Index")
-        print("   events        economic impact events [Finnhub]")
-        print(
-            "   fred          display customized FRED data from https://fred.stlouisfed.org"
-        )
-        print("   vixcls        Volatility Index")
-        print("   gdp           Gross Domestic Product")
-        print("   unrate        Unemployment Rate")
-        print("   dgs1          1-year Treasury Constant Maturity Rate")
-        print("   dgs5          5-year Treasury Constant Maturity Rate")
-        print("   dgs10         10-year Treasury Constant Maturity Rate")
-        print("   dgs30         30-year Treasury Constant Maturity Rate")
-        print("   mortgage30us  30-year Fixed Rate Mortgage Average")
-        print("   fedfunds      Effective Federal Funds Rate")
-        print("   aaa           Moody's Seasoned AAA Corporate Bond Yield")
-        print("   dexcaus       Canada / U.S. Foreign Exchange Rate (CAD per 1 USD)")
-        print("")
-        print("Wall St. Journal:")
-        print("   overview      market data overview")
-        print("   indices       us indices overview")
-        print("   futures       futures overview")
-        print("   usbonds       us bond overview")
-        print("   glbonds       global bonds overview")
-        print("   currencies    currency overview")
-        print("")
-        print(">  report        generate automatic report")
-        print("")
+        help_text = """https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/gamestonk_terminal/economy
+
+>> ECONOMY <<
+
+What do you want to do?
+    cls           clear screen
+    ?/help        show this menu again
+    q             quit this menu, and shows back to main menu
+    quit          quit to abandon program
+CNN:
+    feargreed     CNN Fear and Greed Index
```

Comment:
Not sure if you want to extra line in between the quit and all the sources

Reply:
Yup, good idea!